test/benchmarks/packet.ts

```diff
@@ -39,19 +38,3 @@ const packetEncrypted = moneyPacketV0.serializeAndEncrypt(encryptionKey)
     console.log(this.name, '\t', String(event.target));
   })
   .run({})
-
-;(new Benchmark.Suite('encrypt:     MoneyFrame'))
```

Comment:
Benchmark doesn't support async ?

Reply:
I don't think it does. The encryption/decryption benchmarks aren't really needed anyway. The serialization/deserialization benchmarks are the important ones.